upload-assets/action.yaml

```diff
@@ -0,0 +1,28 @@
+name: "Meticulous - Upload Assets"
+description: "Upload assets to Meticulous for testing."
```

Comment:
I wonder if should name this `upload-assets-report-diffs` as it's the primary entrypoint for executing Meticulous tests (and matches existing 'report diffs' naming)

Reply:
Hmm, but it's already within the `report-diffs-action` repo right? So with the current name they would invoke it as `@alwaysmeticulous/report-diffs-action/upload-assets@v1`. If we re-name it per your suggestion then it would be `@alwaysmeticulous/report-diffs-action/upload-assets-report-diffs@v1` which feels long/duplicative.